oidc_apis/scopes.py

```diff
@@ -85,7 +85,20 @@ class CustomInfoTextStandardScopeClaims(StandardScopeClaims):
     )
 
     def scope_profile(self):
-        return super().scope_profile()
+        dic = {
+            'name': self.userinfo.get('name'),
+            'given_name': (self.userinfo.get('given_name') or
+                           getattr(self.user, 'first_name', None)),
+            'family_name': (self.userinfo.get('family_name') or
+                            getattr(self.user, 'last_name', None)),
+            'middle_name': self.userinfo.get('middle_name'),
+            'nickname': self.userinfo.get('nickname') or getattr(self.user, 'username', None),
```

Comment:
Is it a good idea to pass username as a nickname? In my opinion, it's not and IIRC there's already some code in Tunnistamo to override that behavior of django-oidc-provider.

Reply:
See this comment and code:

https://github.com/City-of-Helsinki/tunnistamo/blob/96fe16a9ed012c43c73ea42e4898aedbc13fb857/tunnistamo/oidc.py#L32-L41

Reply:
@Rikuoja : @suutari-ai has a point. Otherwise, I'm ready to merge this.

Reply:
@suutari-ai actually, the nickname in ```self.userinfo.get('nickname')``` has already been initialized by calling the code you referenced above, so the or-part is never called. However, to be explicit, we might remove the or part from this code too.